### PR TITLE
70 Add API annotations description

### PR DESCRIPTION
**Pull request by Benkoff**
connect to #70

### CODE REVIEW

File: docs/links/swagger.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  "swagger": "2.0",
+  "info": {
+    "title": "Links API",
+    "description": "API for links microservice data exchange.  ",
+    "version": "0.1.0"
+  },
+  "host": "evo.juja.com.ua",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
```
Comment (billMeLater) [inline docs/links/swagger.json]: so far we agreed on http, https is not supported

File: docs/links/swagger.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        "user": {
+          "type": "string",
+          "description": "the user which saved the link"
+        },
+        "url": {
+          "type": "string",
+          "description": "the link been saved before"
+        },
+        "label": {
```
Comment (billMeLater) [inline docs/links/swagger.json]: not aware about label. Dont remember we decided to have that

File: docs/links/swagger.json (hunk 1 above)
Comment (Benkoff) [reply docs/links/swagger.json]: as I can see there is https in fact: https://evo.juja.com.ua

Comment (Benkoff) [reply docs/links/swagger.json]: label removed, thanks. as well as the model names fixed. btw we've got Link model isn't it better to call it LinkRequestResponse? too many Links are bit confusing as for me

**Comment by Benkoff** [comment]
there are typos in the last commit. please read:
it's mostly similar but has some extra descriptions and examples. ~~SaveLinkRequest~~ LinkRequestResponse API model corresponds to Link model in the code, could be renamed later.

**Review by billMeLater — DISMISSED** [review]
looks good

**Comment by Benkoff** [comment]
after our latest decision to use annotations instead of files configuration, those files are deleted and an API annotations description added.
a readable copy: https://github.com/Benkoff/juja-platform/blob/develop/docs/links/api.md

**Review by adaleorion — APPROVED** [review]
thanks

**Pull request merged**